testdata/import/08_import_up.sh

```diff
@@ -2,35 +2,38 @@
 # DO NOT EDIT. Generated by terravalet.
 # WARNING: check the order of resources before running this script.
 #
-# This script will "terraform import" 9 items.
+# This script will "terraform import" 10 items.
 
 # Uncomment this if you want to stop the script at first error
 # set -e
 
 terraform import \
```

Comment:
I tested these changes in up script on 1 real repo by doing the full import cycle.